notebooks/source/model_rendering.ipynb

```diff
@@ -448,32 +448,39 @@
   },
   {
    "cell_type": "markdown",
-   "id": "balanced-command",
+   "id": "50a92902",
    "metadata": {},
    "source": [
-    "## Distribution annotations\n",
-    "\n",
-    "It is possible to display the distribution of each RV in the generated plot by providing `render_distributions=True` when calling `numpyro.render_model`."
+    "## Rendering the parameters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "74f32e20",
+   "metadata": {},
+   "source": [
+    "We can render the parameters defined as `numpyro.param` by setting `render_params=True` in `numpyro.render_model`. "
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
-   "id": "proprietary-baseball",
+   "execution_count": 26,
+   "id": "645df936",
    "metadata": {},
    "outputs": [],
    "source": [
     "def model(data):\n",
-    "    x = numpyro.sample(\"x\", dist.Normal(0, 1))\n",
-    "    y = numpyro.sample(\"y\", dist.LogNormal(x, 1))\n",
+    "    m = numpyro.param(\"m\", jnp.array(0))\n",
+    "    sd = numpyro.param(\"sd\", jnp.array([1]), constraint=constraints.positive)\n",
+    "    lambd = numpyro.sample(\"lambda\", dist.Normal(m, sd))\n",
```

Comment:
I think we can use `dist.LogNormal` to make sure `lambd` is positive.

Reply:
Ohh thanks, I corrected it.